shell/pages/support/index.vue

```diff
@@ -102,8 +99,7 @@ export default {
     },
 
     hasSupport() {
-      // NB: This is temporary until API implemented
-      return false;
+      return isRancherPrime();
```

Comment:
I think we should use `hasSubscriptionSupport` everywhere and get rid of `hasSupport`